frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -75,33 +84,59 @@ export const UnsubscribeSurveyModal = ({
             }
         >
             <div className="flex flex-col gap-3.5">
-                {unsubscribeError ? (
+                {unsubscribeError && (
                     <LemonBanner type="error">
                         <p>
                             {unsubscribeError.detail} {unsubscribeError.link}
                         </p>
                     </LemonBanner>
+                )}
+                {isAddonProduct ? (
+                    <p>We're sorry to see you go! Please note, you'll lose access to the addon features immediately.</p>
                 ) : (
-                    <LemonBanner type="info">
-                        <p>
-                            Any outstanding invoices will be billed immediately.{' '}
-                            <Link to={billing?.stripe_portal_url} target="_blank">
-                                View invoices
-                            </Link>
-                        </p>
-                    </LemonBanner>
+                    <p>
```

Comment:
There's some extra-large margin here due to `p`'s implicit margin-bottom:
```suggestion
                    <p className="mb-0">
```
(this is important so that the modal doesn't require scrolling)